admin/app/components/solidus_admin/orders/index/component.rb

```diff
@@ -143,7 +143,7 @@ def customer_column
       col: { class: "w-[400px]" },
       header: :customer,
       data: ->(order) do
-        customer_email = order.user&.email
+        customer_email = order.order_email
```

Comment:
We should just use `order.email` here this is set to the user's email (either by a logged in user or manually as guest user).